Source/SpatialGDK/Private/SpatialNetDriver.cpp

```diff
@@ -169,6 +169,44 @@ bool USpatialNetDriver::IsLevelInitializedForActor(const AActor* InActor, const
 	return true;
 }
 
+
+void USpatialNetDriver::NotifyActorDestroyed(AActor* ThisActor, bool IsSeamlessTravel /*= false*/)
+{
+	// Remove the actor from the property tracker map
```

Comment:
I think it may be a good idea to temporarily add a `check(!IsSeamlessTravel)` here, so that we trip up when we start doing server travel. Otherwise I'm worried it will be overlooked. Thoughts? @m-samiec @improbable-valentyn